embedded-keycloak/src/main/java/com/playtika/test/keycloak/KeycloakContainer.java

```diff
@@ -0,0 +1,66 @@
+package com.playtika.test.keycloak;
+
+import static com.playtika.test.common.utils.ContainerUtils.containerLogsConsumer;
+import static java.lang.String.format;
+import static java.time.Duration.ofSeconds;
+
+import lombok.extern.slf4j.Slf4j;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.utility.MountableFile;
+
+@Slf4j
+public class KeycloakContainer extends GenericContainer<KeycloakContainer> {
+
+    public static final String KEYCLOAK_USER = "admin";
+    public static final String KEYCLOAK_PASSWORD = "letmein";
+    public static final int HTTP = 8080;
+
+    private final KeycloakProperties properties;
+
+    public KeycloakContainer(KeycloakProperties properties) {
+        super(properties.getDockerImage());
+        this.properties = properties;
+    }
+
+    @Override
+    protected void configure() {
+        withEnv("KEYCLOAK_USER", KEYCLOAK_USER);
+        withEnv("KEYCLOAK_PASSWORD", KEYCLOAK_PASSWORD);
+
+        withCommand(
+            "-c standalone.xml",
+            "-Dkeycloak.profile.feature.upload_scripts=enabled"
+        );
+
+        withStartupTimeout(properties.getTimeoutDuration());
+        withLogConsumer(containerLogsConsumer(log));
+
+        withExposedPorts(HTTP);
+
+        setWaitStrategy(Wait
+            .forLogMessage("(.*)(Keycloak)(.*)(started)(.*)", 1)
+            .withStartupTimeout(ofSeconds(properties.getWaitTimeoutInSeconds()))
+        );
+
+        String importFile = properties.getImportFile();
+        if (importFile != null) {
```

Comment:
It is possible to add only one file?
Also we should throw understandable exception if import files are missing on classpath

Reply:
As I understand the docs only one file is going to work (https://hub.docker.com/r/jboss/keycloak). I think this is fine for the moment. I will add an understandable exception.